package.json

```diff
@@ -176,5 +179,16 @@
 		"test-unit:watch": "npm run test-unit -- --watch",
 		"test-unit-php": "docker-compose run --rm wordpress_phpunit phpunit",
 		"test-unit-php-multisite": "docker-compose run -e WP_MULTISITE=1 --rm wordpress_phpunit phpunit"
+	},
+	"lint-staged": {
+		"packages/*/package.json": [
+			"wp-scripts lint-pkg-json"
+		],
+		"*.js": [
+			"eslint"
```

Comment:
We should rather use `npm run lint-js` in here to ensure it always uses the universal command.

Reply:
@gziolo If we do that, then it will run for all of the JavaScript files and not just the ones that have changed. It is very slow when scanning everything.

Reply:
Essentially the lint staged tool will automatically pass the files that have changed to the command. Because the `npm run lint-js` script passes `.` as the path, it will cause it to run `eslint` against all `.js` files in the project. By just using the `eslint` command here, only the files that have changed are linted. Besides, we don't want linting problems caused by someone else's commit to prevent our own.

Reply:
Unless you use `git add -p` for partially staging files, if you do stage files in this manner `lint-staged` is unable to detect the difference.

See https://github.com/okonet/lint-staged/issues/62

Reply:
_Aside:_ we need `wp-scripts lint-js` anyways, that would be a way to provide one interface in the long run.